src/Symfony/Component/DependencyInjection/Tests/Fixtures/containers/container9.php

```diff
@@ -50,7 +50,7 @@
     ->addMethodCall('setBar', array(new Reference('foo2', ContainerInterface::NULL_ON_INVALID_REFERENCE)))
     ->addMethodCall('setBar', array(new Reference('foo3', ContainerInterface::IGNORE_ON_INVALID_REFERENCE)))
     ->addMethodCall('setBar', array(new Reference('foobaz', ContainerInterface::IGNORE_ON_INVALID_REFERENCE)))
-    ->addMethodCall('setBar', array(new Expression('service("foo").foo() ~ (container.hasparameter("foo") ? parameter("foo") : "default")')))
+    ->addMethodCall('setBar', array(new Expression("service('foo').foo() ~ (container.hasParameter('foo') ? parameter('foo') : 'default')")))
```

Comment:
Fabbot should not check fixtures files actually. It does not make sense. I suggest reverting this change